### DISCUSSION TITLE
Add checksum (for release)

### DISCUSSION THREAD

**Discussion by Rezn1r** [Alex313031/Thorium-Special#70] [Ideas]
# Why?
To provide the integrity of every file
Its simple as getting sha256sum

```
0565695b77108bc19d7e4823c67119c1c2ec797d931ee23435699a79bdfa67b2     file
```

**Comment by gz83** [comment]
thanks for your advice

**Comment by Rezn1r** [comment]
No problem! I requested for this since the AUR Package didn't have a checksum.
